tests/test_code_actions.py

```diff
@@ -100,7 +100,8 @@ def test_no_fix_on_non_matching_kind(self) -> Generator:
         initial_content = 'const x = 1'
         self.view.run_command('lsp_save')
         self.assertEquals(entire_content(self.view), initial_content)
-        self.assertEquals(self.view.is_dirty(), True)
+        yield 100
+        self.assertEquals(self.view.is_dirty(), False)
```

Comment:
Why not do `yield lambda: not self.view.is_dirty()` ?

Reply:
Done